### DISCUSSION TITLE
disable automatic insertion of \item with return inside itemize environment

### DISCUSSION THREAD

**Discussion by pjgrandinetti** [James-Yu/LaTeX-Workshop#3692] [Q&A]
I would like to disable the automatic insertion of "\item" whenever I hit return inside an itemize environment, but I cannot find where to disable it in settings.

**Comment by jlelong** [comment]
See https://github.com/James-Yu/LaTeX-Workshop/wiki/Environments#itemize-like-environments
Set `latex-workshop.bind.enter.key` to `false`

**Reply by pjgrandinetti** [reply]
thanks!
